src/answers/ImagesAnswerComponent.coffee

```diff
@@ -57,12 +57,23 @@ module.exports = class ImagesAnswerComponent extends React.Component
         imagelist = _.map(@props.imagelist or [], (image) -> _.extend({}, image, cover: image.id == id))
         @props.onImagelistChange(imagelist)
 
+      onRotate = () =>
+        @setState(modal: null)
+        imagelist = _.map(@props.imagelist or [], (image) -> 
+          if image.id == id
+            return _.extend({}, image, rotation: (image.rotation + 90) % 360 )
```

Comment:
image.rotation will be undefined for existing images. Which will do weird things when added to 90. Could you write a test for this?